src/util.ts

```diff
@@ -9,6 +9,52 @@ export interface BabelProperty extends ESTree.MethodDefinition {
   decorators?: BabelDecorator[];
 }
 
+/**
+ * Returns if given node has a lit identifier
+ * @param {ESTree.Node} node
+ * @return {boolean}
+ */
+function hasLitIdentifier(
+  node:
+    | ESTree.Node
```

Comment:
this will cause the whole thing to dumb down to `ESTree.Node | null | undefined`

can you change it to consume `ESTree.Node` , and just update the call site further down to null check beforehand?

Reply:
what i mean is you can drop expression and spreadelement too since they're ultimately nodes

`hasLitIdentifier(node: ESTree.Node)` basically

Reply:
applied the changes and works indeed fine.